kb/disorders/Frontotemporal_Dementia.yaml

```diff
@@ -0,0 +1,184 @@
+    preferred_term: Dementia
+    term:
+      id: HP:0000726
+      label: Dementia
+genetic:
+- name: MAPT
+  association: Mutations cause autosomal dominant FTD with tau pathology
+  notes: Microtubule-associated protein tau gene on chromosome 17q21. Mutations lead to FTLD-tau pathology. Patients show earlier onset, prominent behavioral disinhibition, stereotyped behaviors, and temporal lobe predominance of atrophy.
+  evidence:
+  - reference: PMID:26473392
+    supports: SUPPORT
+    snippet: MAPT patients were younger than other groups, and showed more frequent behavioural disinhibition, repetitive and stereotyped behaviours, semantic impairment and temporal predominance of atrophy.
+    explanation: This confirms the distinct clinical and anatomical phenotype associated with MAPT mutations in frontotemporal dementia.
+- name: GRN
+  association: Mutations cause autosomal dominant FTD with TDP-43 pathology
+  notes: Progranulin gene on chromosome 17q21. Haploinsufficiency leads to FTLD-TDP pathology. Patients more likely to present with non-fluent aphasia and are older at symptom onset. GRN and C9orf72 are the most frequent genetic causes.
+  evidence:
+  - reference: PMID:26473392
+    supports: SUPPORT
+    snippet: GRN patients were older at death and more likely to present with non-fluent aphasia.
+    explanation: This demonstrates the characteristic clinical presentation of GRN-related FTD with language deficits.
+  - reference: PMID:30711676
+    supports: SUPPORT
+    snippet: Presymptomatic carriers of GRN and C9orf72 mutations, the most frequent genetic causes of frontotemporal lobar degeneration, represent the optimal target population for the development of disease-modifying drugs.
+    explanation: This confirms GRN as one of the two most common genetic causes of FTD.
+- name: C9orf72
+  association: Hexanucleotide repeat expansion causes autosomal dominant FTD, often with ALS
+  notes: Hexanucleotide (GGGGCC) repeat expansion on chromosome 9p21. Most common genetic cause of FTD. Causes FTLD-TDP pathology. Unique features include co-occurrence with ALS, psychotic symptoms, and paradoxically preserved social warmth.
+  evidence:
+  - reference: PMID:26473392
+    supports: SUPPORT
+    snippet: C9orf72 patients alone showed a co-occurrence of ALS. They showed more psychotic symptoms and irrational behaviour, yet were more often reported clinically as socially appropriate and warm.
+    explanation: This highlights the distinctive clinical features of C9orf72-related FTD including the frequent association with ALS and unique behavioral profile.
+  - reference: PMID:30711676
+    supports: SUPPORT
+    snippet: Presymptomatic carriers of GRN and C9orf72 mutations, the most frequent genetic causes of frontotemporal lobar degeneration, represent the optimal target population for the development of disease-modifying drugs.
+    explanation: This confirms C9orf72 as one of the most common genetic causes of FTD.
+treatments:
```

Comment:
good example of where we might want to have the agent request maxo terms. Also consider NCIT here